programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md

```diff
@@ -19,3 +19,15 @@
 ## Proposed Project ideas
 
 ---
+
+### Vitess
+
+#### Improve Unit Test Coverage
+
+- Description: Vitess is a scalable cloud-native database system for horizontal scaling of MySQL.The project is over 10 years old and there are parts of the code that don’t have very good unit test coverage. Revamping these code files and adding unit test coverage will help with the overall project health. Having strong unit testing is also useful in preventing introducing bugs when making code changes to these files. The task of the mentee would be to add said unit tests for the given code files.
+- Expected Outcome: Improved unit test coverage in Vitess.
```

Comment:
If you know the current coverage, could you please note that down? Or, if you don't know, maybe you would like to state that too, so that measuring the current coverage is also part of the mentorship period.

Reply:
We have code-coverage being run in the project and we display it as a badge in the repo as well. I have included the information in the proposal now 👍